### PR TITLE
Add MATEKF411 target with second soft serial port

### PR DESCRIPTION
**Pull request by cmenard**
This is in response to issue #3919

### CODE REVIEW

File: src/main/target/MATEKF411/target.c [hunk 1]
```diff
@@ -33,8 +33,11 @@ const timerHardware_t timerHardware[] = {
     DEF_TIM(TIM2, CH2, PB3,  TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S5
     DEF_TIM(TIM2, CH3, PB10, TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S6
     DEF_TIM(TIM2, CH1, PA15, TIM_USE_MC_SERVO | TIM_USE_FW_SERVO, 0, 0), // S7
-
+#ifdef MATEKF411_SFTSRL2
+	DEF_TIM(TIM1, CH1, PA8,  TIM_USE_PWM,   0, 0), //softserial_tx2 - 2812LED TIM_USE_LED
```
Comment (digitalentity) [inline src/main/target/MATEKF411/target.c]: Indentation incorrect. Also, this should be set to `TIM_USE_ANY`

Comment (digitalentity) [reply src/main/target/MATEKF411/target.c]: Separate declaration for the PA8 is not needed for this PR. Just change `TIM_USE_LED` to `TIM_USE_ANY`

**Review by digitalentity — CHANGES_REQUESTED** [review]

**Review by digitalentity — APPROVED** [review]
LGTM

**Comment by cmenard** [comment]
Thanks for the feedback!

**Pull request merged**